mmengine/hooks/ema_hook.py

```diff
@@ -165,7 +166,10 @@ def before_save_checkpoint(self, runner, checkpoint: dict) -> None:
         # parameters.
         self._swap_ema_state_dict(checkpoint)
 
-    def after_load_checkpoint(self, runner, checkpoint: dict) -> None:
+    def after_load_checkpoint(self,
+                              runner,
+                              checkpoint: dict,
+                              revise_keys: list = [(r'^module.', '')]) -> None:
```

Comment:
`after_load_checkpoint` is a hook function, its signature should not be changed.